app/sync.js

```diff
@@ -325,7 +324,18 @@ module.exports.init = function (initialState) {
     if (!records || !records.length) {
       return
     }
-    syncUtil.applySyncRecords(records)
+    let showBookmarksToolbar = false
+    if (!initialState.seed) {
+      // syncing for the first time
```

Comment:
on the first sync, i feel like we should just check `records` for any bookmarks and enable the toolbar here